supabase/functions/_shared/helpers.ts

```diff
@@ -13,5 +13,11 @@ export const returnPostgresError = (error: any) => {
 export const handlebarsHelpers = {
   urlencode: function(s: string) {
     return encodeURIComponent(s)
+  },
+  base64encode: function(s: string) {
+    return btoa(s)
```

Comment:
IDK if it matters here, but against all reason the JS `btoa` function will error out on any non-ascii input characters. I'm guessing this is only used in scenarios where we're guaranteed ascii inputs, but worth a check.